src/main/kotlin/me/partlysanestudios/partlysaneskies/commands/PSSCommand.kt

```diff
@@ -0,0 +1,70 @@
+//
+// Written by Su386.
+// See LICENSE for copyright and license notices.
+//
+package me.partlysanestudios.partlysaneskies.commands
+
+import me.partlysanestudios.partlysaneskies.commands.CommandManager.commandList
+import me.partlysanestudios.partlysaneskies.commands.CommandManager.registerCommand
+import net.minecraft.command.ICommand
+import net.minecraft.command.ICommandSender
+
+class PSSCommand(val name: String) {
+    private var aliases: MutableList<String> = ArrayList<String>()
+    private var description: String = ""
+    private var runnable = PSSCommandRunnable { _: ICommandSender?, _: Array<String> -> }
+    var iCommand: ICommand? = null
+    var isRegistered = false
+
+    constructor(
+        name: String,
+        aliases: MutableList<String> ,
+        description: String,
+        runnable: PSSCommandRunnable,
+    ) : this(name, aliases, description) {
+        this.runnable = runnable
+    }
+
+    constructor(name: String, aliases: MutableList<String> , description: String) : this(name, aliases) {
+        this.description = description
+    }
+
+    constructor(name: String, aliases: MutableList<String> ) : this(name) {
+        this.aliases = aliases
+    }
+
+    fun addAlias(alias: String): PSSCommand {
```

Comment:
also add a `fun addAliases(aliases: vararg String)` to make adding multiple aliases easier